setup.py

```diff
@@ -90,7 +90,7 @@ def run(self):
     zip_safe=False,
     platforms="any",
     install_requires=reqs("base.txt") + reqs("docs.txt"),
-    test_requires=reqs("test.txt"),
+    # test_requires=reqs("test.txt"),
```

Comment:
if we don't need this, then we should remove the dead code.

Reply:
Sure will remove this.